besca/_helper.py

```diff
@@ -190,3 +193,98 @@ def get_means(adata,mycat):
         average_obs=None
         fraction_obs=None
     return(average_obs, fraction_obs)
+
+def concate_adata(adata1, adata2):
```

Comment:
How is this different from [AnnData.concatenate](https://anndata.readthedocs.io/en/stable/anndata.AnnData.concatenate.html)

Reply:
@Artemis-R

Reply:
@hatjek From the implementation its similar but it also supports adata objects with .raw objects. This was always an issue for me using the AnnData.concatenate function.